src/networking.c

```diff
@@ -2125,8 +2144,21 @@ sds catClientInfoString(sds s, client *client) {
         if (connHasWriteHandler(client->conn)) *p++ = 'w';
     }
     *p = '\0';
+
+    /* Compute the total memory consumed by this client. */
+    size_t obufmem = getClientOutputBufferMemoryUsage(client);
+    size_t total_mem = obufmem;
+    total_mem += sizeof(client); /* includes client->buf */
```

Comment:
nit: we could also get actual allocation of client rather than sizeof. Probably totally negligible.

Reply:
it might not be negligible, it's likely to have some 30% overhead.

Reply:
30% of just `sizeof(client)` is not that much, but yeah it could be noticeable in some scenarios.

Reply:
note that i fixed that in both CLIENT LIST, and the cron that collects data for INFO MEMORY